src/image_builder.py

```diff
@@ -139,6 +139,27 @@ def image_builder(buildspec):
         if "labels" in image_config:
             labels.update(image_config.get("labels"))
 
+        # Adding standard labels to all images
+        labels[f"com.amazonaws.sagemaker.dlc.framework.{str(BUILDSPEC['framework'])}"] = "true"
```

Comment:
thinking about naming this `com.amazonaws.dlc.sagemaker` - this way we can also have `com.amazonaws.dlc.e3` with the same prefix.